UnitTest/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#Copyright 2022 Ben Saboff
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+project(CPP-Units_Tests)
+
+include(CTest)
+
+set(gtest_foce_shared_crt ON CACHE BOOL "" FORCE)
```

Comment:
Is this needed both here and in the main CMakeList? I would think just here should be sufficient, unless I'm missing something.